README.md

```diff
@@ -220,7 +220,7 @@ We will always endeavor to support the latest stable version of TypeScript. Some
 
 **The version range of TypeScript currently supported by this parser is `>=3.2.1 <3.7.0`.**
 
-This is reflected in the `devDependency` requirement within the package.json file, and it is what the tests will be run against. We have an open `peerDependency` requirement in order to allow for experimentation on newer/beta versions of TypeScript.
+This is reflected in the `devDependency` requirement within the package.json file, and it is what the tests will be run against. We have an open `optionalDependency` requirement in order to allow for experimentation on newer/beta versions of TypeScript.
```

Comment:
maybe would be good to add a comment stating explicitly that we expect the user to install typescript in the project root?

might as well be explicit.